### PR TITLE
Adding r-climate4r packages

### PR DESCRIPTION
**Pull request by cofinoa**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by dbast** [comment]
The PR title speaks about "r-climate4r", but the name in the meta.yaml is "r-transformer"...?

**Comment by cofinoa** [comment]
Yes, this PR will include many other recipes which are part of the climate4r meta-package.

File: recipes/r-transformer/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}git
```
Comment (dbast) [inline recipes/r-transformer/meta.yaml:26]:
```suggestion
```

**Comment by cofinoa** [comment]
@dbast, please, can you review this PR?

This has been a long shot. Thx.

**Comment by cofinoa** [comment]
This PR includes many recipes and probably @dbast it's not anymore interested on review. 

Please @conda-forge/staged-recipes could you take a look?

Thanks in advance.

File: azure-pipelines.yml [hunk 2]
```diff
@@ -12,4 +12,4 @@ pr:
 jobs:
   - template: ./.azure-pipelines/azure-pipelines-linux.yml
   - template: ./.azure-pipelines/azure-pipelines-osx.yml
-  - template: ./.azure-pipelines/azure-pipelines-win.yml
+#  - template: ./.azure-pipelines/azure-pipelines-win.yml
```
Comment (xhochy) [inline azure-pipelines.yml:15]: Please revert the changes to the CI scripts.

**Review by xhochy — CHANGES_REQUESTED** [review]

File: azure-pipelines.yml (hunk 2 above)
Comment (cofinoa) [reply azure-pipelines.yml:15]: Ok. But how I remove the `win` task?

The recipes, and packages, are `noarch` but the `win` task fails at the very begining.

Comment (xhochy) [reply azure-pipelines.yml:15]: Failing `noarch` packages on Windows are OK, we can merge them with a red cross.

**Comment by cofinoa** [comment]
@xhochy I have reverted the CI scripts.

**Comment by xhochy** [comment]
Leaving this for @conda-forge/help-r to review.

**Comment by cofinoa** [comment]
Please, anyone from @conda-forge/help-r to review?

**Comment by cofinoa** [comment]
I haven't received any review from @conda-forge/help-r

Please @conda-forge/staged-recipes can you help?

**Comment by cofinoa** [comment]
@conda-forge-admin, please restart ci

**Comment by cofinoa** [comment]
Closing the PR because no help from @conda-forge/help-r 
I Will re-submit the PR as more atomic R packages

**Pull request closed without merging**